hospital_management_system/hospital_app/models.py

```diff
@@ -1,43 +1,59 @@
 import uuid
 from django.db import models
 from django.contrib.auth.models import User
-
+from rest_framework.authtoken.models import Token as DefaultToken
+from django.core.validators import RegexValidator
+from datetime import datetime
 
 class Hospital(models.Model):
-    models.UUIDField(primary_key = True,default=uuid.uuid4,unique=True,editable=False)
-    name = models.CharField(max_length=200,default='')
-    address = models.TextField(max_length=1000,default='')
-    Staffs_num = models.IntegerField()
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, unique=True, editable=False)
+    name = models.CharField(max_length=200, default='')
+    address = models.TextField(max_length=1000, default='')
+    staffs_num = models.IntegerField()
     num_of_beds = models.IntegerField(unique=True)
```

Comment:
make name or (name+address) PK not beds (gives better option)

Reply:
Fixed